src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -71,11 +71,12 @@ func Run(httpProxyAddr string,
 	beforeStart func(cfg *config.Config) bool,
 	afterStart func(cfg *config.Config),
 	onConfigUpdate func(cfg *config.Config),
+	configFetcher *config.Fetcher,
```

Comment:
This breaks liblantern.  In particular, github.com/getlantern/lantern doesn't pass a configFetcher.